instructor.py

```diff
@@ -1,8 +1,8 @@
 from data import INSTRUCTORS, COURSES
-from color_ui import print_information
 
 class Instructor:
     def __init__(self, instructor_id):
+        self.color_theme = "light"  # Default theme is light
         if instructor_id in INSTRUCTORS:
             self.instructor_id = instructor_id
             self.name = INSTRUCTORS[instructor_id]
```

Comment:
I don't see a place where you use the color_theme? The prompt doesn't change color based on the color_theme.